.gitignore

```diff
@@ -1,3 +1,4 @@
 config.php
+logo.php
```

Comment:
Maybe we should ignore `logo.*` to ignore all potential extensions.

Reply:
Good point, I don't have think at this problem.
I was also thinking about the peering policy file. Should we ignore peering_policy_file.* as well ?

Reply:
It could be a good idea too indeed.